clippy_lints/src/attrs.rs

```diff
@@ -261,7 +261,7 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for Attributes {
                                             _ => {},
                                         }
                                     }
-                                    let line_span = last_line_of_span(cx, attr.span);
+                                    let line_span = first_line_of_span(cx, attr.span);
```

Comment:
This function never returned the last line of the span, but always the first line, so I renamed it accordingly.